usaspending_api/agency/v2/views/agency_overview.py

```diff
@@ -49,3 +51,15 @@ def get_subtier_agency_count(self):
             )
             .count()
         )
+
+    def get_defc(self):
+        return (
+            FinancialAccountsByProgramActivityObjectClass.objects.filter(
+                treasury_account__funding_toptier_agency=self.toptier_agency,
+                disaster_emergency_fund_id__isnull=False,
+                disaster_emergency_fund__group_name="covid_19",
```

Comment:
Since the field is not named after COVID or a specific disaster I would think we want to return all distinct DEFC and let frontend determine if COVID or not based on the DEFC that they see.

Reply:
THat's fair, in the past we've been restricting similar results (like the award summary page) to only covid-related spending, so I assumed it was the same here. @ebdabbs can you weigh in on if the frontend wants all DEFC or only covid-related DEFC?